cluster/ci/config/prow/config.yaml

```diff
@@ -2380,7 +2380,50 @@ presubmits:
         args:
           - ./hack/tf-lint.sh
 
-  - name: ci-pull-openshift-installer-go-vet
+  - name: ci-pull-openshift-installer-shellcheck
+    trigger: "(?m)^/test shellcheck"
+    rerun_command: "/test shellcheck"
+    context: ci/prow/shellcheck
+    always_run: true
+    skip_report: false
+    agent: kubernetes
+    decorate: true
+    spec:
+      containers:
+      - image: quay.io/coreos/shellcheck-alpine:v0.5.0
+        env:
+        - name: IS_CONTAINER
+          value: "TRUE"
+        command:
+        - sh
+        args:
+        - ./hack/shellcheck.sh
+
+  - name: ci-pull-openshift-installer-go-vet-batch1
```

Comment:
I strongly do not suggest doing this, just keep one `go vet` incantation.

Reply:
The job has been updated accordingly